readinglist/__init__.py

```diff
@@ -53,6 +55,7 @@ def set_auth(config):
 
     config.set_authorization_policy(authz_policy)
     config.set_authentication_policy(authn_policy)
+    config.set_default_permission('readwrite')
```

Comment:
Should it be one of the specific permissions used in resource ? Or should it an abstract one like "view", that we override in resource views to readonly/readwrite ?